setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+#Ubuntu 14.04
+#Tested on Digital Ocean's PHPMyAdimin Pre Installed Ubuntu 14.04 server
+
+##IMPORTANT: Inorder to initialize MySQL server load at least one page of the website
+##after the installation.
+
+#https://github.com/kaustubh-karkare/aurora-online-judge
```

Comment:
Add spaces after the `#` at the start of the lines. Comment applies throughout the file.